pkg/xray/resource_xray_security_policy.go

```diff
@@ -51,6 +52,19 @@ func resourceXraySecurityPolicyV2() *schema.Resource {
 				},
 			},
 		},
+		"vulnerability_ids": {
+			Type:        schema.TypeSet,
+			Optional:    true,
+			Set:         schema.HashString,
+			Description: "Creates policy rules for specific vulnerability IDs that you input. You can add multiple vulnerabilities IDs up to 100, separated by \",\". CVEs and Xray IDs are supported. Example - CVE-2015-20107, XRAY-2344",
+			Elem: &schema.Schema{
+				Type:     schema.TypeString,
+				MaxItems: 100,
+				ValidateDiagFunc: validation.ToDiagFunc(
+					validation.StringMatch(regexp.MustCompile(`(CVE-\d\d\d\d-\d+|XRAY-\d+)`), "invalid Vulnerability, must be a valid CVE or Xray ID, example CVE-2021-12345, XRAY-1234"),
```

Comment:
Should we use the regexes suggest here for CVE? https://cve.mitre.org/cve/identifiers/tech-guidance.html#extraction_or_parsing

Reply:
yup, I've missed a minimum of 4 digits.